modules/custom/openy_upgrade_tool/openy_upgrade_tool.routing.yml

```diff
@@ -1,8 +1,31 @@
-openy_upgrade_tool.logger.diff:
-  path: '/admin/config/openy-upgrade-tool/{logger_entity}/diff'
```

Comment:
New diff modal used only `openy_upgrade_log` entity type. No need to support old entity type because it will be migrated in `openy_upgrade_tool_update_8004`

Reply:
If so - eject logger entity into it's own module - OPenY needs it